fake/fake-extension.js

```diff
@@ -830,17 +825,20 @@ var generateMvcRequest = (function() {
                     
                     var payload = message.payload;
                     if (type == 'begin-request') {
-                        request._requestStartTime = payload.requestStartTime;
-                        request._requestMethod = payload.requestMethod;
-                        request._requestUrl = (payload.requestPath || '') + (payload.requestQueryString || '');
+                        payload.path = source.path;
+                        payload.query = source.query;
+                        
+                        request._requestStartTime = payload.startTime;
+                        request._requestMethod = payload.method;
+                        request._requestUrl = (payload.path || '') + (payload.query || '');
                     }
                     if (type == 'end-request') {
-                        request._responseStatusCode = payload.responseStatusCode; 
+                        request._responseStatusCode = payload.statusCode; 
                         request._responseStatusText = source.statusText;
                         request._responseContentCategory = source.contentCategory;  
                         
-                        payload.responseStatusText = request._responseStatusText;  
-                        payload.responseContentCategory = request._responseContentCategory;
+                        payload.statusText = request._responseStatusText;  
+                        payload.contentCategory = request._responseContentCategory;
```

Comment:
The `statusText` and `contentCategory` aren't parts of the message payload spec.  Is this for another purpose?